src/passkey_child/passkey_child_common.c

```diff
@@ -229,6 +250,14 @@ check_arguments(const struct passkey_data *data)
         goto done;
     }
 
+    if (data->action == ACTION_FORMAT_PEM
+        && data->public_key == NULL) {
```

Comment:
Would it be possible to also support passing username/domain on the command line to get the public key?  Or if this is lost does the user simply have to re-register in order to output a new one?

Reply:
Ah right, I forgot we'd have the information in LDAP.   That's all I had.